python/samples/demos/telemetry/scenarios.py

```diff
@@ -22,6 +22,12 @@ def set_up_kernel() -> Kernel:
     # All built-in AI services are instrumented with telemetry.
     # Select any AI service to see the telemetry in action.
     kernel.add_service(OpenAIChatCompletion(service_id="open_ai"))
+    # kernel.add_service(
```

Comment:
Is this code needed?

Reply:
This is for customers to know that they can easily switch between services to run this sample. I also have Google commented out below.

Reply:
Should we add some comment like "The following code shows how to add other AI services to the kernel" ? It just looks like commented out code otherwise.

Reply:
I think we should get in the habit (myself included) of overcommitting the code -- comments help people from all different backgrounds and skillsets.